ideologies.js

```diff
@@ -413,58 +260,22 @@ ideologies = [
             "scty": 5
         }
     },
-    {
-        "name": "Libertarian Capitalism",
-        "stats": {
-            "econ": 20,
-            "dipl": 50,
-            "govt": 80,
-            "scty": 60
-        }
-    },
     {
         "name": "Market Anarchism",
         "stats": {
             "econ": 20,
             "dipl": 50,
-            "govt": 100,
+            "govt": 80,
             "scty": 50
         }
     },
     {
-        "name": "Rational-Egoism",
+        "name": "Objectivism",
         "stats": {
             "econ": 10,
             "dipl": 50,
             "govt": 90,
             "scty": 40
         }
     },
-    {
-        "name": "Totalitarian Capitalism",
-        "stats": {
-            "econ": 0,
-            "dipl": 30,
-            "govt": 0,
-            "scty": 50
-        }
-    },
-    {
-        "name": "Ultra-Capitalism",
-        "stats": {
-            "econ": 0,
-            "dipl": 40,
-            "govt": 50,
-            "scty": 50
-        }
-    },
-    {
-        "name": "Anarcho-Capitalism",

```

Comment:
Dunno why you got rid of these.

Reply:
They all are described by another ideology.